### DISCUSSION TITLE
Could I delete the contents in ".build_cache" folder?

### DISCUSSION THREAD

**Discussion by YYzzzor** [xmake-io/xmake#5459] [Q&A]
Hello developers! 
 When checking the project volume, I found that the folder "path_to_my_project/build/.build_cache" takes up a lot of storage space. Can the contents be deleted?  Does deletion affect the project itself?

**Comment by star-hengxing** [comment]
Yes, you can use `set_policy("build.ccache", false)`
https://xmake.io/#/guide/build_policies?id=buildccache

**Reply by waruqi** [reply]
or xmake f --ccache=n

**Reply by YYzzzor** [reply]
> Yes, you can use `set_policy("build.ccache", false)` https://xmake.io/#/guide/build_policies?id=buildccache

It works. Thank you!

**Reply by YYzzzor** [reply]
> or xmake f --ccache=n

It works. Thank you!
